src/components/TextInputAutoGrow.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import TextInputFocusable from './TextInputFocusable';
+
+const propTypes = {
+
+    // The value of the comment box
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+
+    // A ref to forward to the text input
+    forwardedRef: PropTypes.func.isRequired,
+
+    // General styles to apply to the text input
+    // eslint-disable-next-line react/forbid-prop-types
+    style: PropTypes.any,
```

Comment:
If we are making this `PropTypes.any` then we need to provide support for `PropTypes.any`. Looking at this PR and we are treating this `style` prop as an `Object`.

Reply:
Separately, having `style` and `textStyle` props is ambiguous since there's no way to know without looking at the component which refers to which. If this style is being passed to the input perhaps we should call it `inputStyle` or something more descriptive?